File: pset1/mario/less/mario.c

#include <stdio.h>
#include <cs50.h>

int main(void)
{
	// The height of the pyramid
	int height = 0;

	// Get pyramid height. re-ask if given value isn't valid range
	do {
		printf("Pyramid height: ");
		height = get_int();
	} while (height < 0 || height > 23);

	// Build the pyramid
	for(int i = 0; i < height; i++)
	{
		// Print the spaces within this row
        for(int j = 0; j < height-i-1; j++)
        {
            printf("%s", " ");
        }
        // Print the '#' within this row
		for(int k = 0; k < i+2; k++)
		{
			printf("#");
		}
		printf("\n");
	}
	return 0;
}

File: pset6/mario/mario.py


# Print mario wall
# Author: Jeanniton Monero
# email: jeanniton.mnr@gmail.com
# website: jeanniton.me


def main():

    # The height of the pyramid
    height = -1

    # Get pyramid height. re-ask if given value isn't valid range
    while height < 0 or height > 23:
        print("Pyramid height: ")
        height = (int)(input())

    # Build the pyramid
    for i in range(height):
        # Print the spaces within this row
        for j in range(height-i-1):
            print(" ", end="")
        for k in range(i+2):
            print("#", end="")
        # Go to new row
        print("")


if __name__ == "__main__":
    main()